x-pack/plugins/fleet/common/constants/download_source.ts

```diff
@@ -5,8 +5,9 @@
  * 2.0.
  */
 
-// Default URL used to download Elastic Agent
-export const DEFAULT_DOWNLOAD_SOURCE = 'https://artifacts.elastic.co';
+// Default source URI used to download Elastic Agent
+export const DEFAULT_DOWNLOAD_SOURCE_URI =
+  'https://artifacts.elastic.co/downloads/beats/elastic-agent';
```

Comment:
I replaced the base URI with the complete path as described in the docs [here](https://www.elastic.co/guide/en/fleet/8.3/elastic-agent-reference-yaml.html) under `source_uri` parameter.